### PR TITLE
Add const values for OFPGC_INSERT_BUCKET and OFPGC_REMOVE_BUCKET commands

### PR DESCRIPTION
**Pull request by hongliangl**
To support OFPGC_INSERT_BUCKET and OFPGC_REMOVE_BUCKET commands,
CommandBucketId in GroupMod should be either the bucket ID of one
of the buckets already existing in the group, or one of the
special bucket identifiers, which are added in this PR.

Signed-off-by: Hongliang Liu <lhongliang@vmware.com>

### CODE REVIEW

File: openflow15/group.go [hunk 1]
```diff
@@ -20,6 +20,15 @@ const (
 	 */
 )
 
+const (
+	OFPG_BUCKET_MAX   = 0xffffff00 /* Last usable bucket ID. */
```
Comment (wenyingd) [inline openflow15/group.go:24]:
```suggestion
	OFPG_BUCKET_MAX   = 0xffffff00 /* Max usable bucket ID. */
```

Comment (wenyingd) [reply openflow15/group.go:24]: How to use this constant, e.g., is it only used in OFPGC_INSERT_BUCKET and OFPGC_REMOVE_BUCKET commands?

Comment (hongliangl) [reply openflow15/group.go:24]: It should be set as the value of CommandBucketId in GroupMod when using  OFPGC_INSERT_BUCKET and OFPGC_REMOVE_BUCKET commands.

Comment (hongliangl) [reply openflow15/group.go:24]: This comment is from the openflow file, I think we should reserve the original comment.

/* Bucket Id can be any value between 0 and OFPG_BUCKET_MAX */
enum ofp_group_bucket {
    OFPG_BUCKET_MAX   = 0xffffff00, /* Last usable bucket ID. */
    OFPG_BUCKET_FIRST = 0xfffffffd, /* First bucket ID in the list of action
                                       buckets of a group. This is applicable
                                       for OFPGC_INSERT_BUCKET and
                                       OFPGC_REMOVE_BUCKET commands. */
    OFPG_BUCKET_LAST  = 0xfffffffe, /* Last bucket ID in the list of action
                                          buckets of a group. This is applicable
                                          for OFPGC_INSERT_BUCKET and
                                       OFPGC_REMOVE_BUCKET commands. */
   OFPG_BUCKET_ALL = 0xffffffff  /* All action buckets in a group,
                                     This is applicable for
                                     only OFPGC_REMOVE_BUCKET command. */
};

**Review by wenyingd — APPROVED** [review]

**Pull request merged**